index.html

```diff
@@ -53,6 +53,191 @@ <h4>I’am a Software Developer</h4>
                 </li>     
             </ul>
         </section>
+
+        <section id="recent-works" >
+            <header id="recent-works-header" >
+                <h1>My Recent Works</h1>
+                <div class="thin-line" ></div>
+            </header>           
+            
+            <div id="my-works">
+                <div class="featured" >
+                   
+                    <div class="img-placeholder" ></div>                    
+                    <div class="multi-post">
+                        <h3>Multi-Post Stories</h3>
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text 
+                            ever since the 1500s, when an unknown printer took a standard dummy text.
+                        </p>
+                        <div>
+                            <span class="tags" >css</span>
+                            <span class="tags">html</span>
+                            <span class="tags">Bootstrap</span>
+                            <span class="tags">Ruby</span>
+                        </div>
+                        <button type="button" class="btn btn-feature" >See Project</button>
+                    </div>
+                </div>
+              
+                <ul id="card-container" >
+                    <li class="project-card" >                       
+                        <div class="card-description" >
+                            <h1>Profesional Art Printing Data</h1>
+                            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. 
```

Comment:
- [ ] Please replace `<h1>` tag in your work card header by `<h2>` or `<h3>` tag because you only allowed to use one `<h1>` tag per page for semantic reasons. And please change that for all following work cards.